xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -1151,9 +1290,22 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value indicating whether the collection has a fixed size.</summary>
+        <value>
+          <see langword="true" /> if the collection has a fixed size; otherwise, <see langword="false" />.</value>
```

Comment:
Since this is RO, wouldn't this method return `true`, always?

Reply:
Yeap. Fixed.